.azure-pipelines-templates/ninja.yml

```diff
@@ -3,4 +3,4 @@ steps:
   displayName: Ninja
   workingDirectory: build
   env:
-    CCACHE_DIR: /ccache
+    CCACHE_DIR: /tmp/ccache
```

Comment:
Why this change?

Reply:
ccache tries to create this directory, but when this isn't running in docker the job [doesn't have permission](https://dev.azure.com/MSRC-CCF/CCF/_build/results?buildId=5557&view=logs&j=9be4a519-996c-5dea-a60e-59d3ff2364b1&t=7e4c56d2-1b54-52d4-b692-8716a14d9d94&l=37) to create `/ccache`.